build.gradle

```diff
@@ -379,6 +379,7 @@ project(':iceberg-aws') {
 
     testImplementation 'software.amazon.awssdk:iam'
     testImplementation 'software.amazon.awssdk:s3control'
+    testImplementation 'com.twitter:chill-java:0.10.0'
```

Comment:
should the version number go into `versions.props`?

Reply:
make sense, moved version.props to versions.props, thanks !